python/sglang/lang/ir.py

```diff
@@ -178,10 +182,13 @@ def run(
             stop = []
         if stop_token_ids is None:
             stop_token_ids = []
+        if stop_regex is None:
```

Comment:
nit: maybe we can keep things in the same order. For example, suppose we want the order to be stop - stop_regex - stop_token_ids, then in everywhere we put things in this order.